backends/nxp/tests/test_neutron_backend.py

```diff
@@ -3,7 +3,17 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import torch
+
+from executorch.backends.nxp.backend.edge_program_converter import (
+    EdgeProgramToIRConverter,
+)
+from executorch.backends.nxp.backend.ir.lib.tflite.BuiltinOptions import BuiltinOptions
+from executorch.backends.nxp.backend.ir.lib.tflite.Model import Model
+from executorch.backends.nxp.backend.neutron_target_spec import NeutronTargetSpec
+from executorch.backends.nxp.nxp_backend import PayloadComposer
 from executorch.backends.nxp.tests.executorch_pipeline import to_quantized_edge_program
+from executorch.backends.nxp.tests.executors import graph_contains_any_of_ops
```

Comment:
Jake's intention was to split tests which are importing tensorflow/tflite and which are not - https://github.com/pytorch/executorch/pull/11934 . 

By importing from `executors` module the code automatically imports the tensorflow/tflite too - https://github.com/pytorch/executorch/blob/main/backends/nxp/tests/executors.py#L31 . 

Lets move those tests to `test_neutron_backend_executor` module. 

Sorry for initially incomplete background.